pkg/controller/storagecluster/stork_test.go

```diff
@@ -341,6 +341,10 @@ func testStorkInstallation(t *testing.T, k8sVersionStr string) {
 		expectedSchedDeployment.Spec.Template.Spec.Containers[0].Resources.Requests = nil
 		schedDeployment.Spec.Template.Spec.Containers[0].Resources.Requests = nil
 		require.Equal(t, expectedSchedDeployment.Labels, schedDeployment.Labels)
+		minK8sVersion, _ := version.NewVersion(minSupportedK8sVersion)
+		if k8sVersion.GreaterThanOrEqual(minK8sVersion) {
+			expectedSchedDeployment.Spec.Template.Spec.Containers[0].Image = "k8s.gcr.io/kube-scheduler-amd64:v1.21.0"
+		}
```

Comment:
you don't need this if the test spec yaml is updated to `k8s.gcr.io/kube-scheduler-amd64`